### PR TITLE
OPENEUROPA-1409: Update OE Theme

### PR DESCRIPTION
**Pull request by upchuk**
* Updating the theme in composer.json and lock. 
* Updating config
* Updating code

Let's see if it builds.

### CODE REVIEW

**Comment by upchuk** [comment]
For whoever reviews, I made some comments explaining the changes at code level.

File: behat.yml.dist [hunk 1]
```diff
@@ -128,6 +128,7 @@ site:
         content: "section.main-content"
         language switcher: ".ecl-language-list .ecl-dialog"
         language dialog: ".ecl-dialog"
+        content language block: ".ecl-lang-select-page"
```
Comment (upchuk) [inline behat.yml.dist]: The content language block has been moved on the page, so created a specific region in Behat for it.

File: sites/energy/config/sync/block.block.oe_multilingual_content_language_switcher.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ _core:
   default_config_hash: 25g_t0VuM1bjxTgL69nU29TUgSacREaadlxPUWzD_AY
 id: oe_multilingual_content_language_switcher
 theme: oe_theme
-region: breadcrumb
+region: content
```
Comment (upchuk) [inline sites/energy/config/sync/block.block.oe_multilingual_content_language_switcher.yml]: This config should probably not even be here. But fixed for now by putting in the `content` region of the `oe_theme` theme where it is originally provided in. This config is **not** used anyway as we are using the europa demo theme. It was just preventing proper config import.

File: sites/energy/config/sync/block.block.europa_demo_theme_breadcrumb.yml [hunk 3]
```diff
@@ -1,27 +0,0 @@
-uuid: 6f07fbe9-9853-4497-b6ef-e8f62751e5b2
```
Comment (upchuk) [inline sites/energy/config/sync/block.block.europa_demo_theme_breadcrumb.yml]: the breadcrumb block is removed due to adding the breadcrumb in the page header block in the theme. This is also why the content language block region has been renamed in behat.

File: themes/europa_demo_theme/templates/layout/page.html.twig [hunk 4]
```diff
@@ -9,17 +9,11 @@
 
 {% extends '@oe_theme/layout/page.html.twig' %}
 
-{# @todo remove this customisation after the following issue is resolved #}
-{# @link https://webgate.ec.europa.eu/CITnet/jira/browse/OPENEUROPA-1236 #}
```
Comment (upchuk) [inline themes/europa_demo_theme/templates/layout/page.html.twig]: This was implemented so fixed the page header issue. Left the breadcrumb region in place to have a region to print out the content language block. Not really ok, but for the demo we don't the time now to do it properly.

**Comment by imanoleguskiza** [comment]
We need to figure out what to do with the page language switcher, but other than that the PR is fine.

**Comment by ademarco** [comment]
We have figured out a way to fix the content language switcher on the sites. We will create a follow up and deal with that there so that this can be merged and unblock other issues.

**Pull request merged**